Changelog.md

```diff
@@ -1,5 +1,49 @@
 ## Gazebo Common 4.x
 
+## Gazebo Common 4.7.0 (2023-05-17)
+
+1. Port: 3 -> 4 (#506)
+    * [Pull request #506](https://github.com/gazebosim/gz-common/pull/506)
+
+1. Fix for ffmpeg v6
+    * [Pull request #497](https://github.com/gazebosim/gz-common/pull/497)
+
+1. Include cstring for memcpy
+    * [Pull request #501](https://github.com/gazebosim/gz-common/pull/501)
+
+1. Port: 3 -> 4 (#500) 
```

Comment:
There are a number of 3->4 port PRs here. Can we consolidate them?

Reply:
Whenever I start removing things from the changelog, @scpeters gets on to me ;)

Reply:
Oh, I didn't mean remove them. They could just be listed in a single heading.

Reply:
Updated